libraries/protocol/include/graphene/protocol/asset_ops.hpp

```diff
@@ -49,7 +49,11 @@ namespace graphene { namespace protocol {
       /// When this asset is traded on the markets, this percentage of the total traded will be exacted and paid
       /// to the issuer. This is a fixed point value, representing hundredths of a percent, i.e. a value of 100
       /// in this field means a 1% fee is charged on market trades of this asset.
+      // BSIP81: Asset owners may specify different market fee rate for maker orders and taker orders
+      // After BSIP81 activation, market_fee_percent is the maker fee
       uint16_t market_fee_percent = 0;
+      // After BSIP81 activation, taker_fee_percent is the taker fee
+      uint16_t taker_fee_percent = 0;
```

Comment:
Does new parameter `taker_fee_percent` need to go in struct `additional_asset_options` rather than `asset_options`?

Reply:
Thank you Christopher.  It has been moved in Commit 9cadd21603e860bd5d97bcac13dbeb12eb38121c.